README.md

```diff
@@ -11,19 +11,15 @@
 
 These steps are to be completed by the author of the blog post.
 
-1. Create [an issue](https://github.com/OpenLiberty/blogs/issues/new) for the blog post. This is to help the editors track the progress of the post.
+1. Create an issue for the blog post. This is to help the editors track the progress of the post.
 
    * **All blog posts except release blog posts**
 
-     Create a normal issue. In the description, give a simple outline of the purpose of the blog post. If there is a specific date by which the post must be available, mention that in the description too.
+     Create [a normal issue](https://github.com/OpenLiberty/blogs/issues/new). In the description, give a simple outline of the purpose of the blog post. If there is a specific date by which the post must be available, mention that in the description too.
 
-   * **GA release blog posts** (Open Liberty GA release announcements only)
+   * **GA and beta release blog posts** (Open Liberty GA release announcements only)
 
-     Create an issue using the `Open Liberty release notes` issue template. Make sure to select each task in the issue as you complete it to show progress.
-
-   * **Beta release blog posts** (Open Liberty beta release announcements only)
-
-     Create a normal issue. In the title, make clear that it's for the beta release and which release version.
+     Create [an issue using the `Open Liberty release notes` issue template](https://github.com/OpenLiberty/blogs/issues/new/choose). Make sure to select each task in the issue as you complete it to show progress.
```

Comment:
```suggestion
     Create [an issue using the `Open Liberty GA release blog post` issue template](https://github.com/OpenLiberty/blogs/issues/new/choose). Make sure to select each task in the issue as you complete it to show progress.
```

Update to the latest template name

Reply:
I've changed the name of the template so neither version is correct now. I've updated the text with the new name.